nem-docker/clean-logs.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-# delet elog files older than the 9th logfile 
+# delete elog files older than the 9th logfile
```

Comment:
log

Reply:
ha, I am blind and need to get new glasses. 🤣